tb_ctl.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+SAIC_HOME=.
+TG=ptf
+TARGET=saivs
+ASIC_TYPE=trident2
+
+while getopts "hat" OPT; do
+    case ${OPT} in
+    h)
+        echo "Setup testbed: 1) start client docker; 2) start server docker; 3) create links."
+        echo "Usage: tb_ctl.sh [-a ASIC] [-t TARGET] COMMAND"
+        echo -e "-h\tShow help"
+        echo -e "-a\tSet ASIC. Default $ASIC_TYPE."
+        echo -e "-t\tSet target. Default $TARGET."
+        echo -e "COMMAND\tstart|stop"
+        exit 0
+        ;;
+    a)
+        ASIC_TYPE=$OPTARG
+        echo "ASIC type: ${ASIC_TYPE}"
+        ;;
+    t)
+        TARGET=$OPTARG
+        echo "Target: ${TARGET}"
+        ;;
+    *)
+        echo "Invalid options"
+        exit 1
+        ;;
+    esac
+done
+
+COMMAND=${@: -1}
+[[ ! ${COMMAND} =~ start|stop ]] && {
+    echo "Incorrect COMMAND provided. Allowed: start|stop"
+    exit 1
+}
+
+start_all_ptf() {
```

Comment:
SC uses PTF just as a utility to run the packets. Please rename.

Reply:
ok